rad/prelude/chain-remote.rad

```diff
@@ -0,0 +1,56 @@
+{:module 'prelude/chain-remote
```

Comment:
Can we find a better name for this? Maybe `prelude/guarded-eval`?

Reply:
The idea is to start a file for things that are useful to remote RSMs in general. But I could make it a more specific file. Let me know which you prefer.

Reply:
I’d like to avoid “chain” and “remote” seems kind of redundant to me. But I’d be fine with `machine-remote`.